src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -10,56 +10,35 @@ import useChatCommand from '../../../../utils/chatCommand'
 import styles from './overlay.module.css'
 
 interface OverlayProps {
+    sleeping: boolean,
+    wake: (time: number) => void,
     settings: {
         disableChatPopup: boolean
     }
 }
 export default function Overlay(props: OverlayProps) {
     const [showAmbassadorList, setShowAmbassadorList] = useState(false)
-    const [isOverlayVisible, setIsOverlayVisible] = useState(false)
     const chosenAmbassador = useChatCommand()
     const timeoutRef = useRef<NodeJS.Timeout | undefined>(undefined)
 
     useEffect(() => {
         if(chosenAmbassador !== undefined && props.settings.disableChatPopup === false){
             console.log(props.settings.disableChatPopup)
-            setIsOverlayVisible(true)
             setShowAmbassadorList(true)
-
-            // hide overlay after a few seconds
-            timeoutRef.current = setTimeout(() => {
-                setIsOverlayVisible(false)
-                setShowAmbassadorList(false)
-            }, 6000)
+            timeoutRef.current = setTimeout(() => { setShowAmbassadorList(false) }, 6000)
+            props.wake(6000)
         }
-        return () => clearTimeout(timeoutRef.current as NodeJS.Timeout) 
+        return () => clearTimeout(timeoutRef.current as NodeJS.Timeout)
 
-    }, [chosenAmbassador])
+    }, [chosenAmbassador, props.wake])
 
     useEffect(() => {
-        initMouseEventListener()
-    }, [])
-
-    // create mouse event listener to show/hide overlay if mouse is in the viewport
-    const initMouseEventListener = () => {
-        let body = document.querySelector("body")
-        //check if mouse is in the viewport
-        if(body !== null){
-            body.addEventListener('mouseleave', () => {
-                setIsOverlayVisible(false)
-            })
-        }
-        if(body !== null){
-            body.addEventListener('mouseenter', () => {
-                setIsOverlayVisible(true)
-                clearTimeout(timeoutRef.current as NodeJS.Timeout)
-            })
-        }
-    }
+        if (!props.sleeping) clearTimeout(timeoutRef.current as NodeJS.Timeout)
```

Comment:
I need to revisit this in the morning -- I'm not sure it's doing what I think it does. _Once we've woken up_, we want to stop the auto-dismissal if the user is active. Currently, this will fire when `props.wake` gets called, immediately clearing the timeout.